doc/lua_api.md

```diff
@@ -8704,6 +8710,27 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
     -- If true, item can point to all liquid nodes (`liquidtype ~= "none"`),
     -- even those for which `pointable = false`
 
+    pointabilities = {
+		nodes = {
+			["default:stone"] = "blocking",
+			["group:leaves"] = false,
+		},
+		objects = {
```

Comment:
Reading this, I assume player pointabilities can't be controlled by name, only by armor group?

Reply:
Yes for players only armor group will work.
(I could add a `player:<playername>` functionality but I don't think It's that useful, since the pointabilities get registered before any player can join.)
I would even argue that supporting armor group alone and no object names would be sufficient, since you can always add more groups.
Other things could also be supported for pointabilities, like `drawtypes:<e.g. liquid>`.
So if you think more features are necessary please tell me what I should add.

Reply:
I agree with you that this doesn't need a separate feature, but reading the docs it was unclear for me how and if it works for players. Could you add a sentence to that effect? Something like "player pointabilities can not be controlled by name, only by armor group".

Reply:
Done.